src/statsig/constants.ts

```diff
@@ -81,6 +81,7 @@ export const DynamicConfigs = {
       showTransfers: [networkConfig.defaultNetworkId],
       showWalletConnect: [networkConfig.defaultNetworkId],
       showApprovalTxsInHomefeed: [],
+      showNfts: [networkConfig.defaultNetworkId],
```

Comment:
Is this feature flag usage supposed to be included in this PR?

Reply:
this is not a feature flag, just a new property on the existing multi_chain_features dynamic config

Reply:
Approved the changes to `multi_chain_features` in Statsig.